fluentd/configs.d/openshift/input-pre-prometheus-metrics.conf

```diff
@@ -2,9 +2,9 @@
   @type prometheus
   <ssl>
     enable true
-    certificate_path "/etc/fluent/keys/cert"
-    private_key_path "/etc/fluent/keys/key"
-    ca_path "/etc/fluent/keys/ca"
+    certificate_path "#{ENV['ES_CLIENT_CERT']}"
```

Comment:
Maybe, adding fallback would be nice?
certificate_path "#{ENV['ES_CLIENT_CERT'] || '/etc/fluent/keys/cert'}"
(the same for the other two)

Reply:
IIRC we simply use the env var so that if users wanted to connect to an endpoint that didn't require a cert they could blank it out.

Reply:
Ok.  you have my /lgtm, too.